railties/test/generators/argv_scrubber_test.rb

```diff
@@ -82,9 +82,8 @@ def test_rc_whitespace_separated
         file.puts "--hello --world"
         file.flush
 
-        message = nil
         scrubber = Class.new(ARGVScrubber) {
-          define_method(:puts) { |msg| message = msg }
+          define_method(:puts) { |msg| }
```

Comment:
`define_method(:puts)` is needed for the test?

Reply:
Use the `puts` in `ARGVScrubber#read_rc_file`, so it is necessary to suppress unnecessary message.
https://github.com/rails/rails/blob/68a2888ef08faf38c0643921a8259beef5fbe630/railties/lib/rails/generators/rails/app/app_generator.rb#L595 

Although it is also possible to suppress using `silence_stream`, but since we have already redefined `puts` for the same purpose, I made the same correspondence.
https://github.com/rails/rails/blob/68a2888ef08faf38c0643921a8259beef5fbe630/railties/test/generators/argv_scrubber_test.rb#L121